website/docs/components/button/button-code.md

```diff
@@ -17,7 +20,7 @@ Addons can be installed either by passing the required `tag` to the `addonLeft`/
 
 ## Button with Icon
```

Comment:
I think we can remove this example, the next one already shows how to do it.

Reply:
And the next example also shows that you don't have to nest the icon into the Addon, so it's a bit contradictory